recipes-multimedia/gstreamer/gstreamer1.0_1.16%.bbappend

```diff
@@ -1,3 +1,7 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# SPDX-License-Identifier: LGPL-2.0+
```

Comment:
this is deprecated. please have a look at https://spdx.org/licenses/ we need to use LGPL-2.0-or-later